plugins/poll/test/javascripts/acceptance/polls-test-mobile.js.es6

```diff
@@ -0,0 +1,65 @@
+import { acceptance } from "helpers/qunit-helpers";
+import { clearPopupMenuOptionsCallback } from "discourse/controllers/composer";
+import Fixtures from "fixtures/poll";
+
+acceptance("Rendering polls - mobile", {
+  loggedIn: true,
+  mobileView: true,
+  settings: { poll_enabled: true },
+  beforeEach() {
+    clearPopupMenuOptionsCallback();
+  }
+});
+
+test("Public number poll", async assert => {
+  // prettier-ignore
+  server.get("/t/13.json", () => { // eslint-disable-line no-undef
+    return [200, { "Content-Type": "application/json" }, Fixtures["t/13.json"]];
+  });
+
+  // prettier-ignore
+  server.get("/polls/voters.json", request => { // eslint-disable-line no-undef
+    let body = {};
+
+    if (
+      request.queryParams["post_id"] === "16" &&
```

Comment:
You needn't quote properties in Javascript unless they contain characters that don't work with dot notation. `request.queryParameters.poll_id` is preferred.